config/tag2link.yml

```diff
@@ -0,0 +1,73 @@
+---
+3dmr: https://3dmr.eu/model/$1
+fhrs:id: https://ratings.food.gov.uk/business/en-GB/$1
+ref:bag: https://bagviewer.kadaster.nl/lvbag/bag-viewer/index.html#?searchQuery=$1
+ref:edubase: https://get-information-schools.service.gov.uk/Establishments/Establishment/Details/$1
+ref:ruian:building: https://vdp.cuzk.cz/vdp/ruian/stavebniobjekty/$1
+website: https://web.archive.org/web/*/$1/
+woeid: https://www.flickr.com/places/info/$1
+ref:FR:CEF: http://egliseinfo.catholique.fr/#!lieu:$1
+ref:FR:NAF: https://www.insee.fr/fr/metadonnees/nafr2?q=$1&champRecherche=true
+ref:UAI: http://www.education.gouv.fr/pid24302/annuaire-resultat-recherche.html?lycee_name=$1
+mdb_id: https://molendatabase.nl/nederland/molen.php?nummer=$1
+ref:FR:LaPoste: http://www.laposte.fr/particulier/outils/trouver-un-bureau-de-poste/bureau-detail/$1/$1
+ref:FR:MemorialGenWeb: http://www.memorialgenweb.org/memorial3/html/fr/resultcommune.php?idsource=$1
+contact:yelp: https://www.yelp.com/biz/$1
+flag:wikidata: https://www.wikidata.org/wiki/$1
+ref:FR:museofile: https://www.pop.culture.gouv.fr/notice/museo/$1
+wikidata: https://www.wikidata.org/wiki/$1
+contact:phone: tel:$1
+phone: tel:$1
+image: https://commons.wikimedia.org/wiki/File:$1
+contact:foursquare: https://www.foursquare.com/v/$1
+wikimedia_commons: https://commons.wikimedia.org/wiki/File:$1
+contact:tripadvisor: https://www.tripadvisor.com/$1
+ref:at:gkz: http://www.statistik.at/blickgem/gemDetail.do?gemnr=$1
+mapillary: https://www.mapillary.com/map/im/$1
+ref:FR:SIRET: https://www.infogreffe.fr/societes/entreprise-societe/$1
+de:amtlicher_gemeindeschluessel: http://www.statistik-portal.de/Statistik-Portal/gemeindeverz.asp?G=$1
+twitter: https://twitter.com/$1
+contact:facebook: https://www.facebook.com/$1
+nist:fips_code: http://www.census.gov/quickfacts/table/PST045215/$1,00
+contact:google_plus: https://web.archive.org/web/20190401000000/https://plus.google.com/$1
```

Comment:
Google Plus is dead.

Reply:
All contact:* have been removed (mostly because they don't follow the rules of the wiki)